doc/quickstart/find_data.rst

```diff
@@ -564,6 +564,74 @@ explained in :ref:`extra_facets`, and which content is :download:`available here
 </../esmvalcore/config/extra_facets/ipslcm-mappings.yml>`. These multi-variable
 files must also undergo some data selection.
 
+.. _read_access-esm:
+
+ACCESS-ESM
+^^^^^^^^^^
+
+ESMValTool can read native `ACCESS-ESM <https://research.csiro.au/access/about/esm1-5/>`__
+model output.
+
+.. warning::
+
+  This is the first version of ACCESS-ESM CMORizer for ESMValCore Currently, 
+  only two variables (`tas`, and `pr`) are fully supported.
+
+This is an example of the directory file structure in `config_developer.yml`.
+
+.. code-block:: yaml
+
+  ACCESS:
+    cmor_strict: false
+    input_dir:
+      default:
+        - '{institute}/{sub_dataset}/{exp}/{modeling_realm}/netCDF'
+    input_file:
+      default: '{sub_dataset}.{special_attr}-*.nc'
+    output_file: '{sub_dataset}.{special_attr}'
```

Comment:
```suggestion
    output_file: '{institute}_{sub_dataset}_{special_attr}_{short_name}'
```
Could edit in the `config-developer` file so that the output preprocessed files can have more info like other output files, including short_name and institute and separated by underscores.